### PR TITLE
Add publishing ability to the Threads API

### PR DESCRIPTION
**Pull request by JamelHammoud**
Use like the following:

```
import { ThreadsAPI } from 'threads-api';

const main = async () => {
  const threadsAPI = new ThreadsAPI({
    username: 'jamel.hammoud',
    password: 'password',
  });

  await threadsAPI.publish('Hello World');
};

main();
```

### CODE REVIEW

**Comment by junhoyeo** [comment]
Omg thanks!

File: threads-api/src/threads-api.ts [hunk 1]
```diff
@@ -286,4 +296,101 @@ export class ThreadsAPI {
...
+    const base = 'https://i.instagram.com/api/v1/';
+    const url = `${base}bloks/apps/com.bloks.www.bloks.caa.login.async.send_login_request/`;
+    const blockVersion = '5f56efad68e1edec7801f630b5c122704ec5378adbee6609a448f105f34a9c73';
+
+    const params = encodeURIComponent(
+      JSON.stringify({
+        client_input_params: {
+          password: this.password,
+          contact_point: this.username,
+          device_id: `android-${this.androidId}`,
```
Comment (junhoyeo) [inline threads-api/src/threads-api.ts:314]: Wanna add the `device-` prefix to `this.androidId` itself?
ref: https://github.com/m1guelpf/threads-re#authentication

**Review by junhoyeo — APPROVED** [review]
LGTM. Thank you very much!

File: threads-api/src/threads-api.ts (hunk 1 above)
Comment (JamelHammoud) [reply threads-api/src/threads-api.ts]: Yeah, much better now! I renamed `androidId` to `deviceId` too

**Review by junhoyeo — APPROVED** [review]
Nice!

File: threads-api/src/threads-api.ts (hunk 1 above)
Comment (junhoyeo) [reply threads-api/src/threads-api.ts]: 👍👍👍 Looks great!

**Comment by junhoyeo** [comment]
@JamelHammoud I'll publish the new version after bumping the README. Appreciate your work!

**Comment by junhoyeo** [comment]
@JamelHammoud Just published [v1.1.0](https://github.com/junhoyeo/threads-api/releases/tag/v1.1.0) 🔥 🔥

**Comment by JamelHammoud** [comment]
Awesome!

**Comment by petermazzocco** [comment]
Excited to use this later! Thank you

**Pull request merged**